### DISCUSSION TITLE
If testcase failed , in after all hook maybe find elements undefined

### DISCUSSION THREAD

**Discussion by sinck97** [cypress-io/cypress#14768] [Questions and Help] [answered]
Cannot read property 'length' of undefined Because this error occurred during a `after all` hook we are skipping the remaining tests in the current suite

**Comment by bahmutov** [comment, marked as answer]
There is nothing we can answer for this question, not enough detail.

**Reply by sinck97** [reply]
Details are followed，please do me a favor.

cypress v6.2.0
chrome 88

// / <reference types="Cypress" />
describe('BUG', function () {
    it('testcase', function () {
        cy.visit('https://pan.baidu.com/')
        cy.get('.1').should('have.text','1')
    })

    after(function () {
        cy.visit('https://pan.baidu.com/')
        cy.wait(1000)
        cy.get('.pass-text-input-userName').type('test')
        cy.get('.pass-button-submit[id="TANGRAM__PSP_4__submit"]').click()
        cy.get('.pass-generalError-error').should('have.text','请您输入密码')
    });
})
